builtin/providers/azure/resource_azure_disk.go

```diff
@@ -0,0 +1,301 @@
+package azure
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/svanharmelen/azure-sdk-for-go/management"
+	"github.com/svanharmelen/azure-sdk-for-go/management/virtualmachinedisk"
+)
+
+const dataDiskBlobStorageURL = "http://%s.blob.core.windows.net/disks/%s.vhd"
```

Comment:
This should preferably be exported in the sdk or the below function which uses it below be moved to the sdk.